app.py

```diff
@@ -0,0 +1,22 @@
+from config import Config
+from logger import logger
+from flask import Flask
+from flask import request
+from flask import Response
+
+app = Flask(__name__)
+bot = Config.create_bot()
+
+@app.route('/')
+def hello():
+    return "Hello World!"
+
+@app.route('/msg')
+def msg():
+    text = request.args.get('text', None)
+    channel = request.args.get('channel', None)
+    bot.send_msg(text, channel)
+    return Response(status = 200)
+
+if __name__ == '__main__':
+    app.run(host='0.0.0.0')
```

Comment:
Why not use the config here since you have written  `FLASK_RUN_HOST ` in .env

Reply:
Oh, ya, ok

Reply:
But let me explain here. right now we use flask run, we would not use the code here. flask run would auto check `FLASK_RUN_HOST`